### PR TITLE
update datetime to use pytz

### PR DESCRIPTION
**Pull request by m8ttyB**
For issue #756

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -11,6 +11,7 @@ pyasn1==0.1.7
 pytest==2.3.5
 pytest-mozwebqa
 pytest-xdist==1.8
+pytz===2014.10
```
Comment (davehunt) [inline requirements.txt]: What does the === mean? Do we need to pin this?

File: tests/desktop/test_reviews.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 import pytest
 
 from datetime import datetime
+from pytz import timezone
```
Comment (davehunt) [inline tests/desktop/test_reviews.py]: Nit: We could make the imports PEP8 while we're here. See http://legacy.python.org/dev/peps/pep-0008/#imports

File: requirements.txt (hunk 1 above)
Comment (davehunt) [inline requirements.txt]: FWIW I'm fine with pinning, and just as fine with not pinning. :)

**Comment by davehunt** [comment]
Just a couple of nits, otherwise this looks good to me. Note that I haven't run the tests though (it's late, and I need sleep). r+ assuming the test passes and bonus points for addressing the nits.

File: requirements.txt (hunk 1 above)
Comment (m8ttyB) [inline requirements.txt]: I'll stick to not pinning -- I prefer this too but had seen a potential convention formed on this project.
And oops on the fat-finger with the extra `=`

File: tests/desktop/test_reviews.py (hunk 2 above)
Comment (m8ttyB) [inline tests/desktop/test_reviews.py]: Good nit :-)

**Comment by m8ttyB** [comment]
Thanks @davehunt .. most-excellent nits! @stephendonner r?

**Comment by stephendonner** [comment]
LGTM too, thanks; r+ and a merge-away-we-go

**Pull request merged**